salt/cli/support/collector.py

```diff
@@ -0,0 +1,497 @@
+# coding=utf-8
+from __future__ import absolute_import, print_function, unicode_literals
+import os
+import sys
+import copy
+import yaml
+import json
+import logging
+import tarfile
+import time
+import salt.ext.six as six
+
+if six.PY2:
+    import exceptions
+else:
+    import builtins as exceptions
+    from io import IOBase as file
+
+from io import BytesIO
+
+sys.modules['pkg_resources'] = None
```

Comment:
This looks like it is breaking some tests.

Reply:
Forgot to remove it because of muting some unrelated issues. I've fixed them already here: https://github.com/saltstack/salt/pull/49737